pkg/operator/ceph/object/status_test.go

```diff
@@ -36,4 +36,33 @@ func TestBuildStatusInfo(t *testing.T) {
 	statusInfo := buildStatusInfo(cephObjectStore)
 	assert.NotEmpty(t, statusInfo["endpoint"])
 	assert.Equal(t, "http://rook-ceph-rgw-my-store.rook-ceph:80", statusInfo["endpoint"])
+
+	cephObjectStore = &cephv1.CephObjectStore{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "my-store",
+			Namespace: "rook-ceph",
+		},
+	}
+
+	cephObjectStore.Spec.Gateway.SecurePort = 443
+
+	statusInfo = buildStatusInfo(cephObjectStore)
+	assert.NotEmpty(t, statusInfo["endpoint"])
+	assert.Equal(t, "https://rook-ceph-rgw-my-store.rook-ceph:443", statusInfo["endpoint"])
```

Comment:
Add a line for `assert.Empty(t, statusInfo["secureEndpoint"])`

Reply:
Done.